frontend/railways/railways.html

```
@@ -135,178 +147,192 @@ <h2>Scenic Train Journeys</h2>
       <div class="scenic-grid" id="scenic-container">
         <!-- JS Populated -->
       </div>
- feat/trains-of-india-landing
+      feat/trains-of-india-landing
     </section>
 
-  </footer>
+    </footer>
```

HTMLHint (1.9.2)

[error] Tag must be paired, no start tag: [ </footer> ]

(tag-pair)